bundle/edu.gemini.horizons.server/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala

```diff
@@ -0,0 +1,299 @@
+package edu.gemini.horizons.server.backend
+
+import edu.gemini.spModel.core._
+import org.apache.commons.httpclient.HttpClient
+import org.apache.commons.httpclient.HttpException
+import org.apache.commons.httpclient.NameValuePair
+import org.apache.commons.httpclient.methods.GetMethod
+import edu.gemini.horizons.api.HorizonsException
+import java.io.IOException
+import CgiHorizonsConstants._
+import edu.gemini.horizons.api.HorizonsReply
+import java.util.Date
+import java.text.SimpleDateFormat
+
+import scala.collection.JavaConverters._
+import scala.util.matching.Regex
+import scala.io.Source
+import scalaz._, Scalaz._, scalaz.effect.{ IO, Resource }
```

Comment:
This particular bit of work can't be done in parallel so I wanted to preclude that option by using `IO`, but it's mostly laziness ... the exception and resource handling support in `IO` is missing from `Task` and I wanted to use both here.